### DISCUSSION TITLE
Google Scene Viewer can't show GLTF model exported via three.js .

### DISCUSSION THREAD

**Discussion by goout** [google/model-viewer#3647] [Q&A]
[Scene Viewer](https://arvr.google.com/scene-viewer-preview) shows nothing when i try open that [model.zip](https://github.com/google/model-viewer/files/4677135/model.zip)

No warnings, no errors. That model is result of subtraction operation made with 'three-csg' and exported from three.js via GLTFExporter.

Another viewers like 
[https://gltf-viewer.donmccurdy.com/](https://gltf-viewer.donmccurdy.com/),
[https://sandbox.babylonjs.com/](https://sandbox.babylonjs.com/)
displays model fine.

Three,js too [https://codesandbox.io/s/threejs-typsscript-load-gltf-9e1np?file=/src/client/client.ts](https://codesandbox.io/s/threejs-typsscript-load-gltf-9e1np?file=/src/client/client.ts)

And no any problems with [khronos glTF-Validator/](https://github.khronos.org/glTF-Validator/)

**Comment by goout** [comment]
subtraction operation is nothing to do with it, simple cube created and exported with three js is not displayed too.

[pure cube.zip](https://github.com/google/model-viewer/files/4682011/pure.cube.zip)

**Comment by elalish** [comment]
@tpsiaki Since glTF validator likes it, I presume this is a SceneViewer glTF parsing problem?

**Comment by goout** [comment]
@elalish we found out that problem with non-indexed geometries. 
[discourse.threejs.org](https://discourse.threejs.org/t/gltfexporter-exporting-model-that-show-nothing-at-android-scene-viewer/15519)

**Comment by elalish** [comment]
@tpsiaki Looks like SceneViewer chokes on non-indexed glTFs?
